### ISSUE TITLE
Use the facilities in <boost/timer/timer.hpp> instead

### ISSUE THREAD

**Issue by junghans** [votca/xtp#342]
```
In file included from /tmp/junghans/spack-stage/spack-stage-votca-xtp-1.6_rc1-24mgj3zjwkzycitmnnxbsizxtpmsrcfz/spack-src/src/libxtp/calculators/kmclifetime.cc:21:
/home/junghans/spack.git/opt/spack/linux-centos7-haswell/gcc-8.2.0/boost-1.70.0-z4v74hllqtkbdwbgivc7eoraz7wor4ft/include/boost/progress.hpp:23:70: note: #pragma message: This header is deprecated. Use the facilities in <boost/timer/timer.hpp> instead.
 BOOST_HEADER_DEPRECATED( "the facilities in <boost/timer/timer.hpp>" )
                                                                      ^
```

**Comment by JensWehner** [comment]
@junghans this seems to be the current state of this: any ideas?

https://github.com/boostorg/timer/issues/12

**Comment by JensWehner** [comment]
The other option is just delete the progress bars. Can we ignore that message for the moment?

**Comment by junghans** [comment]
If it is only fixed in the upcoming boost-1.72, let's leave it for now!
